taui/src/components/custom-sign-in.js

```diff
@@ -26,7 +26,7 @@ class SignInHeader extends React.Component {
     return (
       <header className='auth-screen__header auth-header'>
         <h2 className='auth-header__agency'>
-          <img className='auth-header__logo' src='assets/BHAlogo.png' alt='' />
+          <img className='auth-header__logo' src='src/img/echo_combined_logo_STACKED_fullcolor.svg' alt='' />
```

Comment:
```suggestion
          <img className='auth-header__logo' src='assets/echo_combined_logo_STACKED_fullcolor.svg' alt='' />
```
And this file will also need to be added to settings.yml, similar to the one above.